moldesign/geom/symmetry.py

```diff
@@ -83,6 +82,7 @@ def get_symmetrized_coords(self, elem):
         Symmetrize the molecule based on the symmetry operation
         This will work as long as the symmetry operation brings each atom closest to a symmetry relation.
         """
+        import scipy.spatial.distance
```

Comment:
Yeah, you're 100% right that this is bad Python! I decided to do it anyway because `import moldesign` was taking a long time - 5-10 seconds on my machine - because of all the external dependencies getting loaded.

So, I made a bunch of imports lazy by putting them in functions. So, in this example, `scipy` doesn't get imported until until it's first needed. I shielded `openbabel`, `simtk.openmm`, `pyscf`, and `scipy` this way, because they were some of the slowest modules to import.

It would obviously be better if we didn't have to make a tradeoff between import speed and code clarity, of course.